what-changed/model/operation_test.go

```diff
@@ -1310,6 +1311,62 @@ security:
 	assert.Equal(t, ObjectAdded, extChanges.SecurityRequirementChanges[0].Changes[0].ChangeType)
 }
 
+func TestCompareOperations_V3_AddSecurity(t *testing.T) {
+	left := `operationId: coldSecurity
+security: []`
+
+	right := `operationId: coldSecurity
+security:
+  - winter:
+    - cold`
+
+	var lNode, rNode yaml.Node
+	_ = yaml.Unmarshal([]byte(left), &lNode)
+	_ = yaml.Unmarshal([]byte(right), &rNode)
+
+	// create low level objects
+	var lDoc v3.Operation
+	var rDoc v3.Operation
+	_ = low.BuildModel(lNode.Content[0], &lDoc)
+	_ = low.BuildModel(rNode.Content[0], &rDoc)
+	_ = lDoc.Build(lNode.Content[0], nil)
+	_ = rDoc.Build(rNode.Content[0], nil)
+
+	// compare.
+	extChanges := CompareOperations(&lDoc, &rDoc)
+	assert.Equal(t, 1, extChanges.TotalChanges())
+	assert.Equal(t, 0, extChanges.TotalBreakingChanges())
+	assert.Empty(t, extChanges.SecurityRequirementChanges)
```

Comment:
This assert tests the current behavior. Not sure is this is desired.

Reply:
This looks correct to me.